<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChallangeThree</class>
 <widget class="QDialog" name="ChallangeThree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>921</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout"/>
  </widget>
  <widget class="QCheckBox" name="chk_linearisierung">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Linearisierung</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_stretching">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_gammakorrektur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stretching">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stretching</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_gammakorrektur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gammakorrektur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_gammakorrektur">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_stretching">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
